entrypoint.sh

```diff
@@ -3,7 +3,7 @@
 cd "$GITHUB_WORKSPACE"
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-dotenv-linter ${INPUT_DOTENV_LINTER_FLAGS} \
+dotenv-linter --quiet ${INPUT_DOTENV_LINTER_FLAGS} \
```

Comment:
What about using the default input via `action.yml`? :thinking:

Reply:
I also think that having it configurable would be better, is there a reason you've added that flag hardcoded?

Reply:
We should pass `reviewdog` only warnings without additional information, otherwise `reviewdog` won't work.